Microsoft.Identity.Web/Microsoft.Identity.Web.csproj

```diff
@@ -57,6 +57,6 @@
     <PackageReference Include="Microsoft.AspNetCore.Authentication.AzureAD.UI" Version="2.2.0" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.AzureADB2C.UI" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.2.0" />
-    <PackageReference Include="Microsoft.Identity.Client" Version="4.3.1" />
+    <PackageReference Include="Microsoft.Identity.Client" Version="4.5.1-internal20191022.1isappcache" />
```

Comment:
Experimental. Contains PR https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1448, which not yet merged